primitives/manta/src/types.rs

```diff
@@ -51,7 +51,7 @@ pub type CalamariAssetId = u128;
 pub type Balance = u128;
 
 /// Transaction Index Type
-pub type Index = u32;
+pub type Nonce = u32;
```

Comment:
It might affect frontend. I don't know. We will see.

Reply:
Yeah this renaming goes along with the change in `frame_system`